packages/ui/react-ui-editor/src/extensions/link.ts

```diff
@@ -92,16 +89,12 @@ export const link = (options: LinkOptions = {}): Extension => {
 };
 
 class LinkButton extends WidgetType {
-  constructor(
-    private readonly _pos: number,
-    private readonly _url: string,
-    private readonly _onAttach: LinkOptions['onRender'],
-  ) {
+  constructor(private readonly _url: string, private readonly _onAttach: LinkOptions['onRender']) {
     super();
   }
 
   override eq(other: LinkButton) {
-    return this._pos === other._pos && this._url === other._url;
+    return this._url === other._url;
```

Comment:
if there were multiple links on the same page with the same URL, this would still be fine?

Reply:
Yes. The widgets just display the URL. They'd be interchangeable.